components/src/core/components/Pagination/Pagination.vue

```diff
@@ -195,7 +195,7 @@ export default defineComponent({
 });
 </script>
 
-<style src="./pagination.scss" lang="scss" scoped></style>
+<style src="./pagination.scss" lang="scss"></style>
```

Comment:
SCSS should be as scoped within the component and otherwise, it'll leak outside.

Reply:
corrected